docs/BindingWithGrammar.md

```diff
@@ -0,0 +1,138 @@
+# Binding with Grammar
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides support for automatic XML schema/grammar binding through generation and/or using an existing `.xsd/.dtd` file.
+
```

Comment:
Here I will show directly what looks like the support mean. In other words, I will add:

Here a demo which show how to bind an XML file with an existing XSD file. (and you add a demo with Codelens which open a xsd and bind it).